Dockerfile

```diff
@@ -0,0 +1,35 @@
+# To build image:
+#
+#     % docker build --tag fam:latest .
+#
+# To run an existing playbook simply specify the path. Override any variables necessary using the
+# --extra-vars option.
+#
+#     % docker run -it --rm \
+#          fam:latest \
+#              --extra-vars "foreman_server_url=https://192.168.122.3" \
+#              /foreman-ansible-modules/test/test_playbooks/organization.yml
+#
+# To run a custom playbook use the --volume option to mount it in. If the playbook references
+# other resources it needs to be mounted relative to those files.
+#
+#     % docker run -it --rm \
+#          --volume `pwd`/myorg.yml:/foreman-ansible-modules/test/test_playbooks/myorg.yml \
+#          fam:latest \
+#              --extra-vars "foreman_server_url=https://192.168.122.3" \
+#              /foreman-ansible-modules/test/test_playbooks/myorg.yml
+
+
+FROM fedora
+
+RUN dnf -y update && dnf install -y \
+        git \
+        python-pip
+
+RUN set -x \
```

Comment:
could we maybe add a set -e as well?
I like adding it to almost all bash scripts I am running. Makes it a usually a bit more stable.

Reply:
I may not be understanding the -e option, but is it relevant in a build? The "set -x" I added based upon another Dockerfile written by more skilled folks (skopeo) and some googling. I'm happy to add it but would like to understand the why.

Reply:
I agree it might not be that useful here since it's using ```&&``` meaning the whole ```RUN``` statement should fail if one fails.

Reply:
yes, that is absolutely true. 
never mind, I was typing quicker than thinking :P Friday.. ;)